index.js

```diff
@@ -727,21 +689,29 @@
       [$.Function([a, b, c, d, e, r]), a, b, c, d, e, r],
       curry5);
 
-  //# flip :: ((a, b) -> c) -> b -> a -> c
+  //# flip :: (a -> b -> c) -> b -> a -> c
   //.
-  //. Takes a binary function and two values, and returns the result of
-  //. applying the function to the values in reverse order.
+  //. Takes a curried binary function and two values, and returns the
+  //. result of applying the function to the values in reverse order.
   //.
-  //. See also [`C`](#C).
+  //. This is the `C` combinator from combinatory logic
```

Comment:
Three minor points:

  - In this context C should not be rendered as code. The description of `I`, for example, simply states:

    > The I combinator.

  - The sentence should end with a full stop (`.`).

  - We don't mention combinatory logic in the descriptions of the other combinators, but it seems reasonable to do so here given that `flip` is not listed in the __Combinator__ section.

I suggest:

_This is the C combinator from combinatory logic._